persistence-typed/src/main/scala/org/apache/pekko/persistence/typed/state/exception/DurableStateStoreExceptions.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.persistence.typed.state.exception
```

Comment:
I think this should lives in the `persistence` package, instead the `persistence.typed`

Reply:
I'm travelling for a few days. Later this week, I will refactor this PR to try to move the new exceptions back to the main pekko-persistence lib instead of the typed one.

Reply:
refactored - please have a look when you have time